client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx

```diff
@@ -118,69 +78,19 @@ const ContactQuestioning: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Eleme
                         allContactedInteractions.sort((firstInteractedContact, secondInteractedContact) =>
                         firstInteractedContact.phoneNumber ? firstInteractedContact.phoneNumber.localeCompare(secondInteractedContact.phoneNumber) : 0).map((interactedContact) => (
                                 <div key={interactedContact.id} className={classes.form}>
-                                    <Accordion expanded={interactedContact.expand} className={classes.accordion} style={{ borderRadius: '3vw'}}>
+                                    <Accordion className={classes.accordion} style={{ borderRadius: '3vw'}}>
```

Comment:
Why style?

Reply:
Didn't change it, but doesn't work in class.